x-pack/plugins/ingest_manager/server/saved_objects.ts

```diff
@@ -148,6 +148,8 @@ export const savedObjectMappings = {
   },
   [PACKAGES_SAVED_OBJECT_TYPE]: {
     properties: {
+      name: { type: 'keyword' },
```

Comment:
I added the name property to the saved object so I could search for the package name to see if its installed.  I added version because I think we will need it do check versions at some point.  I didn't like the idea of parsing the saved object id for name and version.